lib/clusterbuster/workloads/files.workload

```diff
@@ -119,8 +118,8 @@ function files_process_options() {
     fi
 }
 
-function files_namespace_policy() {
-    echo privileged
+function files_requires_drop_cache() {
+    :
 }
```

Comment:
I think you should add it on separate PR

Reply:
This is closely related to the pod privilege change (the drop cache requires a privileged pod) and simplifies that code.